mmtrack/models/task_modules/track/interpolation.py

```diff
@@ -86,6 +132,11 @@ def interpolate_tracks(tracks: np.ndarray,
                                                     max_num_frames)
         else:
             interpolated_track = track
+
+        if gsi:
+            interpolated_track = gaussian_smoothed_interpolation(
+                interpolated_track, smooth_tau)
```

Comment:
Are you sure that you first use linear interpolation, then use gaussian interpolation in your original code?

Reply:
yes, you can find it in https://github.com/dyhBUPT/StrongSORT/blob/b6e981c94448c9d3be7f718ef23ebada1324ec3e/GSI.py#L63-L67